ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -1981,8 +1971,8 @@ private bool SetEnumerable(object collectionObject)
         private void ClearMerged()
         {
             List<IXLRange> mergeToDelete;
-            using (var asRange = AsRange())
-                mergeToDelete = Worksheet.Internals.MergedRanges.Where(merge => merge.Intersects(asRange)).ToList();
+            var asRange = AsRange();
+            mergeToDelete = Worksheet.Internals.MergedRanges.Where(merge => merge.Intersects(asRange)).ToList();
```

Comment:
Remove `asRange` variable and use only `mergeToDelete = Worksheet.Internals.MergedRanges.Where(merge => merge.Intersects(AsRange())).ToList();`